test/integration/tasks/delete-stale-entries.ts

```diff
@@ -0,0 +1,42 @@
+import { EntryProps, KeyValueMap } from 'contentful-management/types'
+import { plainClient } from '../contentful-client'
+import { printStepTitle } from '../utils'
+
+const TWO_HOURS_IN_MS = 60 * 60 * 2 * 1000
+
+export default async (client = plainClient) => {
+  printStepTitle('Removing stale entries from master-test')
```

Comment:
What about `test-base`? Curious if those can lead to piling up as you had to delete nearly 1000 of them

Reply:
The entries I manually deleted were produced by an error in copy-entries.
That was resolved by https://github.com/contentful/ui-extensions-sdk/commit/4b4e073bb25628620a724b9725a986cadb526065#diff-ddc7cc8d83237e60e8073e30613fdff8e7a89cf5247c6a096d8f7a77f31955b5